compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangNodeTransformer.java

```diff
@@ -4231,6 +4221,76 @@ private BLangMatchPattern transformMatchPattern(Node matchPattern) {
         return null;
     }
 
+    private BLangBindingPattern transformBindingPattern(Node bindingPattern) {
+        Location pos = getPosition(bindingPattern);
+        SyntaxKind patternKind = bindingPattern.kind();
+        switch (patternKind) {
+            case WILDCARD_BINDING_PATTERN:
```

Comment:
Similarly here, I would suggest to have each switch case logic in separate methods and call from here. That way, it is more readable.

Reply:
Fixed.